keychain_haiku.cpp

```diff
@@ -0,0 +1,129 @@
+/******************************************************************************
+ *   Copyright (C) 2011-2015 Frank Osterfeld <frank.osterfeld@gmail.com>      *
```

Comment:
Maybe you want to change that line.

Reply:
Added a fixup commit, I'll rebase later on.